### PR TITLE
Adding JPNTP pool 

### PR DESCRIPTION
**Pull request by yoshis2**
Cool to become a part of the Single Pool Alliance

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -3871,5 +3871,19 @@
     "telegram": "w_o_u_t",
     "operatorName": "Wout",
     "github": "tiligallery"
+  },
+  {
+    "ticker": "JPNTP",
+    "poolId": "3d9acdf1c60be1eb069e3abe2235e0cbaf302463183eb0f6a986e302",
+    "social": {
+      "website": "https://crypto-currency-academy.com/",
+      "twitter": "https://twitter.com/xrpripple7",
```
Comment (Bez625) [inline registry.json:3880]: Please can you change this to just `xrpripple7`?

File: registry.json [hunk 2]
```diff
@@ -3871,5 +3871,19 @@
...
+      "facebook": "",
+      "instagram": "",
+      "youtube": ""
+    },
+    "telegram": "",
```
Comment (Bez625) [inline registry.json:3885]: You must have a telegram handle to join the alliance. This must be the telegram of an individual and not a group or channel

File: registry.json [hunk 3]
```diff
@@ -3871,5 +3871,19 @@
...
+    "operatorName": "yoshis2",
```
Comment (Bez625) [inline registry.json:3886]: Please change this to be either your real full name, real first name or leave blank

**Review by Bez625 — CHANGES_REQUESTED** [review]

**Comment by yoshis2** [comment]
Pointers have been corrected.

**Pull request merged**